### PR TITLE
fix: list objects null pointer if computed userset does not exist

### PR DESCRIPTION
**Pull request by craigpastro**
## Description
fix: list objects null pointer if computed userset does not exist

## References
https://github.com/openfga/openfga/issues/571

## Review Checklist
- [x] The correct base branch is being used, if not `main`
- [x] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: internal/graph/graph.go [hunk 1]
```diff
@@ -196,6 +196,9 @@ func (g *ConnectedObjectGraph) findIngressesWithRewrite(
 		for _, typeRestriction := range tuplesetTypeRestrictions {
 
 			r, err := g.typesystem.GetRelation(typeRestriction.GetType(), computedUserset)
+			if err != nil {
+				continue // skip computed relations on tupleset relationships if they are undefined
```
Comment (jon-whit) [inline internal/graph/graph.go:200]: We should probably be more specific and check specifically on `typesystem.ErrRelationUndefined`

```
if errors.Is(err, typesystem.ErrRelationUndefined) {
    continue
}

return nil, err
```

Comment (jon-whit) [reply internal/graph/graph.go:200]: Otherwise we might blindly skip something that should have led to an error otherwise.

Comment (craigpastro) [reply internal/graph/graph.go]: Good point! Updated. Thanks!

File: assets/tests/listobjects_1_0_tests.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+tests:
```
Comment (miparnisari) [inline assets/tests/listobjects_1_0_tests.yaml:1]: TODO 😆

**Review by jon-whit — APPROVED** [review]

**Pull request merged**